### PR TITLE
DockerHub push action on new release tag

### LINKED ISSUE #2181: Set up a webhook to build and push a new Docker image to Docker hub every time the repo changes
I'm not sure how to do it, need to google

### PR DESCRIPTION
**Pull request by northwestwitch**
Builds the Docker image and tags with it repo version tag and "latest", Pushes both images to Docker Hub. 
fix #2181

**How to test**:
1. Locally I'll try on a forked repo, and I'll push to my user's Hub

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested by creating a release tagged as 4.26 from my fork of Scout (after setting DOCKER_USERNAME and DOCKER_PASSWORD secrets in my repo). It works:

image

**Comment by northwestwitch** [comment]
@moonso if you're sitting there wondering how to spend the next 5 minutes of your life, this one needs a reviewer 😆

File: .github/workflows/build_and_publish.yml [hunk 1]
```diff
@@ -38,3 +38,20 @@ jobs:
...
+    - name: Check out git repository
+      uses: actions/checkout@v2
+
+    - name: Build and publish to Docker Hub
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: clinicalgenomics/scout
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        tags: "latest,${{ github.event.release.tag_name }}"
```
Comment (moonso) [inline .github/workflows/build_and_publish.yml:56]: Nice! I will copy this, never really understood how to use both tags

**Review by moonso — APPROVED** [review]
😎

File: .github/workflows/build_and_publish.yml (hunk 1 above)
Comment (northwestwitch) [reply .github/workflows/build_and_publish.yml:56]: Perhaps you need to change also the action source. I'm using one different from yours here.

**Pull request merged**